src/pickers/NeDateFormatter.js

```diff
@@ -1,14 +1,13 @@
-import NepaliDate from 'nepali-date-converter'
+import NepaliDate from "nepali-date-converter";
 
-export function formatDateFromISO(date) {
-  const secondCalendarFormat = ['DD MM YYYY', 'np']
+export function formatDateFromISO(date, formattingOptions) {
   if (!date) return "";
   // this nepali-date-converter only supports dates up to this date, it will crash whole site if this date is exceeded
   // added as a safeguard, other solution is probably needed
   // TODO remove after nepal component change
-  if (date > new Date("2090/12/30")){
+  if (date > new Date("2090/12/30")) {
```

Comment:
This "2090/12/30" could be also moved to the seperate const.

Reply:
I will leave it as it is, it is a temporary solution, comment explains it, besides it will likely be changed to a different solution soon.